snapshot-cross-account-replicator/lambda_source_account.tf

```diff
@@ -20,7 +20,7 @@ resource "aws_lambda_function" "step_1" {
 resource "aws_cloudwatch_event_rule" "invoke_step_1_lambda" {
   provider            = aws.source
   description         = "Triggers the lambda function ${aws_lambda_function.step_1.function_name}"
-  schedule_expression = var.snapshot_schedule_expression
+  schedule_expression = "rate(${var.snapshot_schedule_period} hours)"
```

Comment:
This would be a breaking change to the current functionality. Though for our own internal use cases, it's ok.

Reply:
> I assume that by internal, you mean Skyscrapers and our customers?

Yes (considering we also publish this module publicly)